src/App.less

```diff
@@ -12,10 +12,17 @@ body,
 label {
   background-color: @ffe-farge-fjell;
   color: @ffe-farge-sand;
-  font-family: Cantarell;
   font-size: 20px;
 }
 
+h1,
+h2,
+h3,
+h4,
```

Comment:
Risky å style på elementet, når hvert spill har hver sin layout

Reply:
Jeg kan gå gjennom appen, men tror vi sjekket at det ikke ødela for noen spill (og at hovedfargen da bør være hvit, kan heller overstyre der det trengs). Men skal jobbe videre med denne neste torsdag tenker jeg :-)